rust/otap-dataflow/crates/otap/src/syslog_cef_receiver.rs

```diff
@@ -249,20 +322,27 @@ impl local::Receiver<OtapPdata> for SyslogCefReceiver {
                                                 _ = interval.tick() => {
                                                     if arrow_records_builder.len() > 0 {
                                                         // Build the Arrow records and send them
+                                                        let items = u64::from(arrow_records_builder.len());
                                                         let arrow_records = arrow_records_builder.build().expect("Failed to build Arrow records");
 
                                                         // Reset the builder for the next batch
                                                         arrow_records_builder = ArrowRecordsBuilder::new();
 
-                                                        if let Err(_e) = effect_handler.send_message(OtapPdata::new_todo_context(arrow_records.into())).await {
-                                                            return; // Break out of the entire task
+                                                        let res = effect_handler.send_message(OtapPdata::new_todo_context(arrow_records.into())).await;
+                                                        {
+                                                            let mut m = metrics.borrow_mut();
+                                                            match &res {
+                                                                Ok(_) => m.received_logs_forwarded.add(items),
+                                                                Err(_) => m.received_logs_forward_failed.add(items),
+                                                            }
                                                         }
+                                                        if res.is_err() { return; }
```

Comment:
Note for myself to remove this for the same reason as https://github.com/open-telemetry/otel-arrow/pull/1309#discussion_r2449866092